src/js/redux/reducers/dashboard/dashboardFilterReducer.js

```diff
@@ -6,113 +6,123 @@
-                endDate: ''
-            }
-        },
-        certified: {
-            agencies: [],
-            fileNames: [],
-            submissionIds: [],
-            createdBy: [],
-            lastModified: {
-                startDate: '',
-                endDate: ''
-            }
-        }
+  dabs: {
+    active: {
+      agencies: [],
+      fileNames: [],
+      submissionIds: [],
+      createdBy: [],
+      lastDateModified: {
+        startDate: '',
+        endDate: ''
+      }
     },
-    fabs: {
-        active: {
-            agencies: [],
-            fileNames: [],
-            submissionIds: [],
-            createdBy: [],
-            lastModified: {
-                startDate: '',
-                endDate: ''
-            }
-        },
-        published: {
-            agencies: [],
-            fileNames: [],
-            submissionIds: [],
-            createdBy: [],
-            lastModified: {
-                startDate: '',
-                endDate: ''
-            }
-        }
+    certified: {
+      agencies: [],
+      fileNames: [],
+      submissionIds: [],
+      createdBy: [],
+      lastDateModified: {
+        startDate: '',
+        endDate: ''
+      }
     }
+  },
+  fabs: {
+    active: {
+      agencies: [],
+      fileNames: [],
+      submissionIds: [],
+      createdBy: [],
+      lastDateModified: {
+        startDate: '',
+        endDate: ''
+      }
+    },
+    published: {
+      agencies: [],
+      fileNames: [],
+      submissionIds: [],
+      createdBy: [],
+      lastDateModified: {
+        startDate: '',
+        endDate: ''
+      }
+    }
+  }
 };
 
 export const dashboardFilterReducer = (state = initialState, action) => {
-    switch (action.type) {
-        case 'UPDATE_DASHBOARD_FILTER': {
-            const table = Object.assign({}, state[action.dashboard][action.table], {
-                [action.filter]: action.value
-            });
+  switch (action.type) {
+    case 'UPDATE_DASHBOARD_FILTER': {
+      const table = Object.assign({}, state[action.dashboard][action.table], {
+        [action.filter]: action.value
+      });
 
-            const dashboard = Object.assign({}, state[action.dashboard], {
-                [action.table]: table
-            });
+      const dashboard = Object.assign({}, state[action.dashboard], {
+        [action.table]: table
+      });
 
-            return Object.assign({}, state, {
-                [action.dashboard]: dashboard
-            });
+      return Object.assign({}, state, {
+        [action.dashboard]: dashboard
+      });
+    }
+    case 'TOGGLE_DASHBOARD_FILTER': {
+      // get the existing list
+      let list;
+      if (!Array.isArray(state[action.dashboard][action.table][action.filter])) {
+        const Objectlist = action.value;
+        const prevFilter = state[action.dashboard][action.table][action.filter];
+        if (prevFilter.startDate === Objectlist.startDate && prevFilter.endDate === Objectlist.endDate) {
```

Comment:
`TOGGLE_DASHBOARD_FILTER` is designed to update filters that are stored as arrays of strings.  Instead of modifying it to fit the needs of the date modified filter, you could use `UPDATE_DASHBOARD_FILTER` to store the object with a start date and end date, or create a new action.